cookbook/sonoff-basic-light-switch.rst

```diff
@@ -0,0 +1,161 @@
+DIY Light switch using a Sonoff Basic
+=====================================
+
+.. seo::
+    :description: An example of how to integrate a light switch into Home Assistant using ESPHome
+    :image: sonoff_light_switch.png
+    :keywords: Relay, Sonoff Basic, Sonoff Dual Dual R1, Light, HASS, Home Assistant, ESPHome
+
+.. note::
+
+  This is a DIY solution, and you will need to have some knowledge of electrical wiring and enough
+  capabilities to do this work safely.
+
+  The author, and the ESPHome team, take no responsibility for any actions, injuries or outcomes
+  from following this guide.
+
+  In some countries you may need specific qualifications before you can carry out such work in
+  a residentaial property.
+
+Background
+----------
+
+I was looking for a simple, cost effective, way to upgrade my lights so they could be controlled by
```

Comment:
The ESPHome docs are not using the personal form "I" - with the exception of the changelog, but that has an author set in the SEO tags. Besides, the reader can't know who "I" refers to in this case. If it's somewhere else too that's a mistake.

Please change this to general (I don't know what to call it) style. Example:

-> The idea of this project is to find a simple, cost effective, way to [...]

Reply:
Of course, my mistake. I will correct that tomorrow.

Reply:
I made those changes a while ago now for you, any other changes you want to make this PR happen?